docs/config.json

```diff
@@ -900,8 +900,12 @@
           "slug": "/desktop-access/getting-started/"
         },
         {
-          "title": "Manual Setup",
-          "slug": "/desktop-access/manual-setup/"
+          "title": "Active Directory",
```

Comment:
I'm not sure this title is descriptive enough now. Maybe `Set up with Active Directory`?

Reply:
Note that this is the title displayed in the sidebar, not the title of the document.

Reply:
Ah, no concerns in that case then.